math/fityk/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           github    1.0
 PortGroup           wxWidgets 1.0
 PortGroup           boost     1.0
 
-github.setup        wojdyr fityk 1.3.1 v
+github.setup        wojdyr fityk 1.3.2 v
 revision            0
 name                fityk
```

Comment:
this should already be se from the github.setup line

Reply:
Sorry, overlooked that.